Java/src/main/java/airport/Airport.java

```diff
@@ -77,63 +76,43 @@ public List<MilitaryPlane> getBomberMilitaryPlanes() {
 
     }
 
-    public List<experimentalPlane> getExperimentalPlanes() {
-        List<experimentalPlane> experimentalPlanes = new ArrayList<>();
+    public List<ExperimentalPlane> getExperimentalPlanes() {
+        List<ExperimentalPlane> experimentalPlanes = new ArrayList<>();
         for (Plane plane : planes) {
-            if (plane instanceof experimentalPlane) {
-                experimentalPlanes.add((experimentalPlane) plane);
+            if (plane instanceof ExperimentalPlane) {
```

Comment:
Please reformat all classes and remove some extra empty lines, like this:
<img width="981" alt="image" src="https://user-images.githubusercontent.com/90613721/182925510-e6d3c80b-6d05-4722-ac2e-63a896b10495.png">

Reply:
Were deleted all extra empty lines